server/sonar-server/src/main/java/org/sonar/server/component/index/ComponentIndexQuery.java

```diff
@@ -29,12 +29,18 @@
 
 public class ComponentIndexQuery {
 
+  public enum Sort {
+    NAME, SCORE
+  }
```

Comment:
It sounds interesting ! But in this case, I would move the definition of Sort in ComponentIndex instead of ComponentIndexQuery, WDYT ?